ui/pages/reports/index.html

```diff
@@ -35,10 +35,13 @@
         <h5>Export data from Prometheus as CSV, YAML, and more.</h5>
         <button id="openModalBtn">Export Data</button>
     </div>
+
+    <div id="loadingIndicator" class="loading-indicator"></div>
+
     <div id="exportModal" class="modal">
         <div class="modal-content">
             <span class="close">&times;</span>
-            <h2>Export Data from Prometheus</h2>
+            <h2>Export Data from Promethes</h2>
```

Comment:
```suggestion
            <h2>Export Data from Prometheus</h2>
```
can you please revert this change?